rules/Php73/Rector/FuncCall/StringifyStrNeedlesRector.php

```diff
@@ -99,7 +93,7 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        if ($this->nodeTypeAnalyzer->isStringyType($needleType)) {
+        if ($needleType->isString()->yes()) {
```

Comment:
phpstan know way more string types then the NodeTypeAnalyzer recognized. lets phpstan decide what a string is